src/pool.rs

```diff
@@ -0,0 +1,104 @@
+use crate::client::Client;
+use crate::cmd::{Command, Get, Set};
+use crate::Result;
+use bytes::Bytes;
+use std::time::Duration;
+use tokio::sync::mpsc::{unbounded_channel, UnboundedReceiver, UnboundedSender};
+use tokio::sync::oneshot::{channel, Sender as OneshotSender};
+use tracing::error;
+
+/// create a new connection Pool from a Client
+pub fn create(client: Client) -> Pool {
+    let (tx, rx) = unbounded_channel();
```

Comment:
We should avoid unbounded buffering in the examples.

I would throw up a bounded channel hard coded to 32 or something. In a real-app, the buffer size would be configurable, but we don't need to do that here.